search/domain/service.go

```diff
@@ -36,13 +36,22 @@ type (
 		SortOptions    []SortOption
 	}
 
-	// SortOption defines how sorting is possible, with both an asc and desc option
+	// SortOption defines how sorting is possible, and which of them are activated with both an asc and desc option
 	SortOption struct {
-		Label        string
-		Asc          string
-		Desc         string
-		SelectedAsc  bool
+		// Label that you normally want to show in the frontend (e.g. "Price")
+		Label string
+		// Field that you need to use in SearchRequest>SortFilter
+		Field string
+		// SelectedAsc true if sorting by this field is active
+		SelectedAsc bool
+		// SelectedDesc true if sorting by this field is active
 		SelectedDesc bool
+		// Asc - represents the field that is used to trigger ascending search.
+		// Deprecated: use "Field" and "SelectedAsc" instead to set which field should be sortable
```

Comment:
The message is wrong
It should say: Use "Field" to know which Field to set in the SortFilter